internal/rpc/rpcserver/server.go

```diff
@@ -2152,6 +2162,60 @@ func (s *loaderServer) checkReady() bool {
 	return atomic.LoadUint32(&s.ready) != 0
 }
 
+// StartAccountMixerService starts the AccountMixerService.
+func StartAccountMixerService(server *grpc.Server, loader *loader.Loader) {
+	accountMixerService.loader = loader
+	if atomic.SwapUint32(&accountMixerService.ready, 1) != 0 {
+		panic("service already started")
+	}
+}
+
+// RunAccountMixer starts the automatic account mixer for the service.
+func (t *accountMixerServer) RunAccountMixer(req *pb.RunAccountMixerRequest, svr pb.AccountMixerService_RunAccountMixerServer) error {
+	wallet, ok := t.loader.LoadedWallet()
+	if !ok {
+		return status.Errorf(codes.FailedPrecondition, "Wallet has not been loaded")
+	}
+
+	tb := ticketbuyer.New(wallet)
+
+	// Set ticketbuyerV2 config
+	tb.AccessConfig(func(c *ticketbuyer.Config) {
+		c.MixedAccountBranch = req.MixedAccountBranch
+		c.MixedAccount = req.MixedAccount
+		c.ChangeAccount = req.ChangeAccount
+		c.CSPPServer = req.CsppServer
+		c.BuyTickets = false
```

Comment:
Seems you also need to specify `MixChange = true`.

After doing these changes and using the server jrick specified the mixes are being done.